gems/pending/Scvmm/miq_hyperv_disk.rb

```diff
@@ -114,13 +114,23 @@ def bread_cached(start_sector, number_sectors)
         length           = sectors_in_range * @block_size
         remaining_blocks = number_sectors - sectors_in_range
         @cache_hits[start_sector] += 1
-        return @block_cache[block_range][buffer_offset, length] + bread_cached(block_range.last + 1, remaining_blocks)
+        return @block_cache[block_range][buffer_offset, length] << bread_cached(block_range.last + 1, remaining_blocks)
```

Comment:
Is this concatenating two strings or appending to an Array...very difficult to understand this